### ISSUE TITLE
Update 6.1.0 to 7.0.0 - ConcurrentModificationException crashs

### ISSUE THREAD

**Issue by GaelCO** [realm/realm-java#6876]
Hi,

Since I have updated to realm 7.0.0 (previoulsly 6.1.0), some ConcurrentModificationException crashs happen.
I don't see any code changes that could explain this crash, except for the realm SDK update.

Unlikely, I have never reproduct this issue myself. I get this exception by crashlytics. So I can't give you the reproduct steps.

As you have a better knowledge of the Realm SDK than me, I expect you to help me find out the source of this issue.

The error indicates a concurrent modification, but can you tell me if there are some kinds of modifications that have more probability to provoke this crash?

#### Version of Realm and tooling
Realm version: 7.0.0
Realm Sync feature enabled: No
Android Studio version: 3.6.3
Android Build Tools version: 29.0.3
Gradle version: 6.4.1

Trace :+1: Fatal Exception: java.util.ConcurrentModificationException
       at java.util.HashMap$HashIterator.nextNode(HashMap.java:1441)
       at java.util.HashMap$EntryIterator.next(HashMap.java:1475)
       at java.util.HashMap$EntryIterator.next(HashMap.java:1473)
       at io.realm.internal.ColumnIndices.refresh(ColumnIndices.java:121)
       at io.realm.RealmSchema.refresh(RealmSchema.java:260)
       at io.realm.BaseRealm$1.onSchemaChanged(BaseRealm.java:97)
       at io.realm.internal.Table.nativeFreeze(Table.java)
       at io.realm.internal.Table.freeze(Table.java:676)
       at io.realm.internal.OsResults.freeze(OsResults.java:324)
       at io.realm.RealmResults.freeze(RealmResults.java:542)
       at io.realm.rx.RealmObservableFactory$6$1.onChange(RealmObservableFactory.java:196)
       at io.realm.rx.RealmObservableFactory$6$1.onChange(RealmObservableFactory.java:192)
       at io.realm.internal.ObservableCollection$RealmChangeListenerWrapper.onChange(ObservableCollection.java:39)
       at io.realm.internal.ObservableCollection$CollectionObserverPair.onChange(ObservableCollection.java:20)
       at io.realm.internal.ObservableCollection$Callback.onCalled(ObservableCollection.java:64)
       at io.realm.internal.ObservableCollection$Callback.onCalled(ObservableCollection.java:54)
       at io.realm.internal.ObserverPairList.foreach(ObserverPairList.java:109)
       at io.realm.internal.OsResults.notifyChangeListeners(OsResults.java:638)
       at android.os.MessageQueue.nativePollOnce(MessageQueue.java)
       at android.os.MessageQueue.next(MessageQueue.java:326)
       at android.os.Looper.loop(Looper.java:181)
       at android.os.HandlerThread.run(HandlerThread.java:65)

**Comment by cmelchior** [comment]
It looks like an RxJava stream from a query result somehow resulting in this. I'm a bit puzzled why that would result in this error though. I would probably be trivially fixed by synchronizing `RealmSchema.refresh()` but I would like to understand better exactly what is going on.

**Comment by nongdenchet** [comment]
Hi @cmelchior I am facing similar issue with 7.0.1

Realm version: 7.0.1
Realm Sync feature enabled: No
Android Studio version: 4.0
Android Build Tools version: 29.0.3
Gradle version: 6.1.1

```
2020-07-03 10:30:59.286 23418-23418/? A/DEBUG: signal 6 (SIGABRT), code -6 (SI_TKILL), fault addr --------
2020-07-03 10:30:59.286 23418-23418/? A/DEBUG: Abort message: 'JNI DETECTED ERROR IN APPLICATION: JNI NewLocalRef called with pending exception java.util.ConcurrentModificationException: 
        at java.util.HashMap$Node java.util.HashMap$HashIterator.nextNode() (HashMap.java:1441)
        at java.util.Map$Entry java.util.HashMap$EntryIterator.next() (HashMap.java:1475)
        at java.lang.Object java.util.HashMap$EntryIterator.next() (HashMap.java:1473)
        at void io.realm.internal.ColumnIndices.refresh() (ColumnIndices.java:121)
        at void io.realm.RealmSchema.refresh() (RealmSchema.java:260)
        at void io.realm.BaseRealm$1.onSchemaChanged() (BaseRealm.java:97)
        at long io.realm.internal.Table.nativeFreeze(long, long) (Table.java:-2)
        at io.realm.internal.Table io.realm.internal.Table.freeze(io.realm.internal.OsSharedRealm) (Table.java:676)
        at io.realm.internal.OsResults io.realm.internal.OsResults.freeze(io.realm.internal.OsSharedRealm) (OsResults.java:324)
        at io.realm.RealmResults io.realm.RealmResults.freeze() (RealmResults.java:542)
        at void io.realm.rx.RealmObservableFactory$6$1.onChange(io.realm.RealmResults) (RealmObservableFactory.java:196)
        at void io.realm.rx.RealmObservableFactory$6$1.onChange(java.lang.Object) (RealmObservableFactory.java:192)
        at void io.realm.internal.ObservableCollection$RealmChangeListenerWrapper.onChange(java.lang.Object, io.realm.OrderedCollectionChangeSet) (ObservableCollection.java:39)
        at void io.realm.internal.ObservableCollection$CollectionObserverPair.onChange(java.lang.Object, io.realm.internal.OsCollectionChangeSet) (ObservableCollection.java:20)
        at void io.realm.internal.ObservableCollection$Callback.onCalled(io.realm.internal.ObservableCollection$CollectionObserverPair, java.lang.Object) (ObservableCollection.java:64)
        at void io.realm.internal.ObservableCollection$Callback.onCalled(io.realm.internal.ObserverPairList$ObserverPair, java.lang.Object) (ObservableCollection.java:54)
        at void io.realm.internal.ObserverPairList.foreach(io.realm.internal.ObserverPairList$Callback) (ObserverPairList.java:109)
        at void io.realm.internal.OsResults.notifyChangeListeners(long) (OsResults.java:638)
        at void android.os.MessageQueue.nativePollOnce(long, int) (MessageQueue.java:-2)
        at android.os.Message android.os.MessageQueue.next() (MessageQueue.java:336)
        at void android.os.Looper.loop() (Looper.java:197)
        at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:7811)
        at java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[]) (Method.java:-2)
        at void com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run() (RuntimeInit.java:493)
        at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:1076)
    
        in call to NewLocalRef
        from void android.os.MessageQueue.nativePollOnce(long, int)'
2020-07-03 10:30:59.286 23418-23418/? A/DEBUG:     x0  0000000000000000  x1  0000000000005365  x2  0000000000000006  x3  0000007fccf55680
2020-07-03 10:30:59.287 23418-23418/? A/DEBUG:     x4  7400000000000000  x5  7400000000000000  x6  7400000000000000  x7  0000000000008074
2020-07-03 10:30:59.287 23418-23418/? A/DEBUG:     x8  00000000000000f0  x9  7f6e1fb0b242d724  x10 0000000000000001  x11 0000000000000000
2020-07-03 10:30:59.287 23418-23418/? A/DEBUG:     x12 fffffff0fffffbdf  x13 0000000000000002  x14 0000000000000000  x15 000049f9c8ea3177
2020-07-03 10:30:59.287 23418-23418/? A/DEBUG:     x16 0000007de714d8c0  x17 0000007de7129fe0  x18 00000000ca62c1d6  x19 0000000000005365
2020-07-03 10:30:59.287 23418-23418/? A/DEBUG:     x20 0000000000005365  x21 00000000ffffffff  x22 0000007c638cb000  x23 0000007d618baa40
2020-07-03 10:30:59.287 23418-23418/? A/DEBUG:     x24 0000007d6189a9ac  x25 0000007d61dd8000  x26 0000007de79bc258  x27 0000007d61dd8000
2020-07-03 10:30:59.287 23418-23418/? A/DEBUG:     x28 0000000000000043  x29 0000007fccf55720
2020-07-03 10:30:59.287 23418-23418/? A/DEBUG:     sp  0000007fccf55660  lr  0000007de70db27c  pc  0000007de70db2a8
2020-07-03 10:30:59.548 23418-23418/? A/DEBUG: backtrace:
2020-07-03 10:30:59.548 23418-23418/? A/DEBUG:       #00 pc 00000000000832a8  /apex/com.android.runtime/lib64/bionic/libc.so (abort+160) (BuildId: 55ce0a7d78144b0290f9746ed1615719)
2020-07-03 10:30:59.548 23418-23418/? A/DEBUG:       #01 pc 00000000004ba0fc  /apex/com.android.runtime/lib64/libart.so (art::Runtime::Abort(char const*)+2388) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.548 23418-23418/? A/DEBUG:       #02 pc 000000000000b458  /system/lib64/libbase.so (android::base::LogMessage::~LogMessage()+580) (BuildId: e71ff76c9d3a6b93309ae5824b646459)
2020-07-03 10:30:59.548 23418-23418/? A/DEBUG:       #03 pc 00000000004ff660  /apex/com.android.runtime/lib64/libart.so (art::Thread::AssertNoPendingException() const+1136) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.548 23418-23418/? A/DEBUG:       #04 pc 0000000000166a48  /apex/com.android.runtime/lib64/libart.so (art::ClassLinker::FindClass(art::Thread*, char const*, art::Handle<art::mirror::ClassLoader>)+64) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.548 23418-23418/? A/DEBUG:       #05 pc 0000000000153a48  /apex/com.android.runtime/lib64/libart.so (art::ClassLinker::DoResolveType(art::dex::TypeIndex, art::Handle<art::mirror::DexCache>, art::Handle<art::mirror::ClassLoader>)+184) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.548 23418-23418/? A/DEBUG:       #06 pc 000000000052714c  /apex/com.android.runtime/lib64/libart.so (_ZN3art8verifier4impl12_GLOBAL__N_114MethodVerifierILb0EE6VerifyEv$09bf97eb2c0d684adb6d215f7005036e+1540) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.548 23418-23418/? A/DEBUG:       #07 pc 000000000052acc0  /apex/com.android.runtime/lib64/libart.so (art::verifier::MethodVerifier::FindLocksAtDexPc(art::ArtMethod*, unsigned int, std::__1::vector<art::verifier::MethodVerifier::DexLockInfo, std::__1::allocator<art::verifier::MethodVerifier::DexLockInfo>>*, unsigned int)+648) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.548 23418-23418/? A/DEBUG:       #08 pc 000000000040f7ac  /apex/com.android.runtime/lib64/libart.so (art::Monitor::VisitLocks(art::StackVisitor*, void (*)(art::ObjPtr<art::mirror::Object>, void*), void*, bool)+820) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.548 23418-23418/? A/DEBUG:       #09 pc 0000000000410744  /apex/com.android.runtime/lib64/libart.so (art::MonitorObjectsStackVisitor::VisitFrame()+164) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.548 23418-23418/? A/DEBUG:       #10 pc 00000000004dcc38  /apex/com.android.runtime/lib64/libart.so (_ZN3art12StackVisitor9WalkStackILNS0_16CountTransitionsE0EEEvb+1644) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.548 23418-23418/? A/DEBUG:       #11 pc 00000000004fe3c4  /apex/com.android.runtime/lib64/libart.so (art::Thread::DumpJavaStack(std::__1::basic_ostream<char, std::__1::char_traits<char>>&, bool, bool) const+452) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.548 23418-23418/? A/DEBUG:       #12 pc 00000000004fa234  /apex/com.android.runtime/lib64/libart.so (art::Thread::DumpStack(std::__1::basic_ostream<char, std::__1::char_traits<char>>&, bool, BacktraceMap*, bool) const+536) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #13 pc 000000000051487c  /apex/com.android.runtime/lib64/libart.so (art::DumpCheckpoint::Run(art::Thread*)+828) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #14 pc 000000000050da04  /apex/com.android.runtime/lib64/libart.so (art::ThreadList::RunCheckpoint(art::Closure*, art::Closure*)+592) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #15 pc 000000000050ce60  /apex/com.android.runtime/lib64/libart.so (art::ThreadList::Dump(std::__1::basic_ostream<char, std::__1::char_traits<char>>&, bool)+1964) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #16 pc 00000000004b9d54  /apex/com.android.runtime/lib64/libart.so (art::Runtime::Abort(char const*)+1452) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #17 pc 000000000000b458  /system/lib64/libbase.so (android::base::LogMessage::~LogMessage()+580) (BuildId: e71ff76c9d3a6b93309ae5824b646459)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #18 pc 00000000003786e8  /apex/com.android.runtime/lib64/libart.so (art::JavaVMExt::JniAbort(char const*, char const*)+1584) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #19 pc 000000000037890c  /apex/com.android.runtime/lib64/libart.so (art::JavaVMExt::JniAbortV(char const*, char const*, std::__va_list)+108) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #20 pc 000000000036ad34  /apex/com.android.runtime/lib64/libart.so (art::(anonymous namespace)::ScopedCheck::AbortF(char const*, ...)+136) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #21 pc 0000000000369808  /apex/com.android.runtime/lib64/libart.so (art::(anonymous namespace)::ScopedCheck::CheckPossibleHeapValue(art::ScopedObjectAccess&, char, art::(anonymous namespace)::JniValueType)+1144) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #22 pc 0000000000368bf0  /apex/com.android.runtime/lib64/libart.so (art::(anonymous namespace)::ScopedCheck::Check(art::ScopedObjectAccess&, bool, char const*, art::(anonymous namespace)::JniValueType*)+652) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #23 pc 000000000036b17c  /apex/com.android.runtime/lib64/libart.so (art::(anonymous namespace)::CheckJNI::NewRef(char const*, _JNIEnv*, _jobject*, art::IndirectRefKind)+664) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #24 pc 00000000001561b4  /data/app/com.invoice2go.dev-LgL_VeShbFkqNjzLoIl9ZA==/lib/arm64/librealm-jni.so (BuildId: e40ca2fe74c1a4778f371d2fb07ae692cc14e133)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #25 pc 0000000000156134  /data/app/com.invoice2go.dev-LgL_VeShbFkqNjzLoIl9ZA==/lib/arm64/librealm-jni.so (BuildId: e40ca2fe74c1a4778f371d2fb07ae692cc14e133)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #26 pc 00000000001518cc  /data/app/com.invoice2go.dev-LgL_VeShbFkqNjzLoIl9ZA==/lib/arm64/librealm-jni.so (BuildId: e40ca2fe74c1a4778f371d2fb07ae692cc14e133)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #27 pc 0000000000193dc0  /data/app/com.invoice2go.dev-LgL_VeShbFkqNjzLoIl9ZA==/lib/arm64/librealm-jni.so (BuildId: e40ca2fe74c1a4778f371d2fb07ae692cc14e133)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #28 pc 000000000018cc24  /data/app/com.invoice2go.dev-LgL_VeShbFkqNjzLoIl9ZA==/lib/arm64/librealm-jni.so (BuildId: e40ca2fe74c1a4778f371d2fb07ae692cc14e133)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #29 pc 0000000000177f00  /data/app/com.invoice2go.dev-LgL_VeShbFkqNjzLoIl9ZA==/lib/arm64/librealm-jni.so (BuildId: e40ca2fe74c1a4778f371d2fb07ae692cc14e133)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #30 pc 0000000000198e9c  /data/app/com.invoice2go.dev-LgL_VeShbFkqNjzLoIl9ZA==/lib/arm64/librealm-jni.so (BuildId: e40ca2fe74c1a4778f371d2fb07ae692cc14e133)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #31 pc 000000000019a76c  /data/app/com.invoice2go.dev-LgL_VeShbFkqNjzLoIl9ZA==/lib/arm64/librealm-jni.so (BuildId: e40ca2fe74c1a4778f371d2fb07ae692cc14e133)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #32 pc 000000000001836c  /system/lib64/libutils.so (android::Looper::pollInner(int)+832) (BuildId: 1f16b1d0bb5561a2ecdcbb4fa60a0b61)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #33 pc 0000000000017f8c  /system/lib64/libutils.so (android::Looper::pollOnce(int, int*, int*, void**)+56) (BuildId: 1f16b1d0bb5561a2ecdcbb4fa60a0b61)
2020-07-03 10:30:59.549 23418-23418/? A/DEBUG:       #34 pc 000000000016c4b0  /system/lib64/libandroid_runtime.so (android::android_os_MessageQueue_nativePollOnce(_JNIEnv*, _jobject*, long, int)+44) (BuildId: a65bb63f108cccc2155253f97153af87)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #35 pc 00000000002d034c  /system/framework/arm64/boot-framework.oat (art_jni_trampoline+140) (BuildId: 49cb6ce0cbaf010d2726747535cdcb7239878f72)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #36 pc 000000000205acd4  /memfd:/jit-cache (deleted) (android.os.MessageQueue.next+180)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #37 pc 0000000000137334  /apex/com.android.runtime/lib64/libart.so (art_quick_invoke_stub+548) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #38 pc 0000000000145fec  /apex/com.android.runtime/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+244) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #39 pc 00000000002e3d6c  /apex/com.android.runtime/lib64/libart.so (art::interpreter::ArtInterpreterToCompiledCodeBridge(art::Thread*, art::ArtMethod*, art::ShadowFrame*, unsigned short, art::JValue*)+384) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #40 pc 00000000002defcc  /apex/com.android.runtime/lib64/libart.so (bool art::interpreter::DoCall<false, false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, art::JValue*)+892) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #41 pc 00000000005a1bac  /apex/com.android.runtime/lib64/libart.so (MterpInvokeVirtual+648) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #42 pc 0000000000131814  /apex/com.android.runtime/lib64/libart.so (mterp_op_invoke_virtual+20) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #43 pc 000000000039a766  /system/framework/framework.jar (android.os.Looper.loop+130)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #44 pc 00000000005a491c  /apex/com.android.runtime/lib64/libart.so (MterpInvokeStatic+1040) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #45 pc 0000000000131994  /apex/com.android.runtime/lib64/libart.so (mterp_op_invoke_static+20) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #46 pc 00000000001a7c24  /system/framework/framework.jar (android.app.ActivityThread.main+208)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #47 pc 00000000002b5080  /apex/com.android.runtime/lib64/libart.so (_ZN3art11interpreterL7ExecuteEPNS_6ThreadERKNS_20CodeItemDataAccessorERNS_11ShadowFrameENS_6JValueEbb.llvm.7038959461964381116+240) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #48 pc 0000000000593114  /apex/com.android.runtime/lib64/libart.so (artQuickToInterpreterBridge+1032) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #49 pc 0000000000140468  /apex/com.android.runtime/lib64/libart.so (art_quick_to_interpreter_bridge+88) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.550 23418-23418/? A/DEBUG:       #50 pc 00000000001375b8  /apex/com.android.runtime/lib64/libart.so (art_quick_invoke_static_stub+568) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #51 pc 000000000014600c  /apex/com.android.runtime/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+276) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #52 pc 00000000004b1884  /apex/com.android.runtime/lib64/libart.so (art::(anonymous namespace)::InvokeWithArgArray(art::ScopedObjectAccessAlreadyRunnable const&, art::ArtMethod*, art::(anonymous namespace)::ArgArray*, art::JValue*, char const*)+104) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #53 pc 00000000004b3428  /apex/com.android.runtime/lib64/libart.so (art::InvokeMethod(art::ScopedObjectAccessAlreadyRunnable const&, _jobject*, _jobject*, _jobject*, unsigned long)+1480) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #54 pc 000000000043e1c8  /apex/com.android.runtime/lib64/libart.so (art::Method_invoke(_JNIEnv*, _jobject*, _jobject*, _jobjectArray*)+52) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #55 pc 00000000000c2c54  /system/framework/arm64/boot.oat (art_jni_trampoline+180) (BuildId: 02fb151c7c0ffacd83709e852752a7df4ca81f02)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #56 pc 0000000000137334  /apex/com.android.runtime/lib64/libart.so (art_quick_invoke_stub+548) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #57 pc 0000000000145fec  /apex/com.android.runtime/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+244) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #58 pc 00000000002e3d6c  /apex/com.android.runtime/lib64/libart.so (art::interpreter::ArtInterpreterToCompiledCodeBridge(art::Thread*, art::ArtMethod*, art::ShadowFrame*, unsigned short, art::JValue*)+384) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #59 pc 00000000002defcc  /apex/com.android.runtime/lib64/libart.so (bool art::interpreter::DoCall<false, false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, art::JValue*)+892) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #60 pc 00000000005a1bac  /apex/com.android.runtime/lib64/libart.so (MterpInvokeVirtual+648) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #61 pc 0000000000131814  /apex/com.android.runtime/lib64/libart.so (mterp_op_invoke_virtual+20) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #62 pc 000000000048fb1a  /system/framework/framework.jar (com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run+22)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #63 pc 00000000002b5080  /apex/com.android.runtime/lib64/libart.so (_ZN3art11interpreterL7ExecuteEPNS_6ThreadERKNS_20CodeItemDataAccessorERNS_11ShadowFrameENS_6JValueEbb.llvm.7038959461964381116+240) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #64 pc 0000000000593114  /apex/com.android.runtime/lib64/libart.so (artQuickToInterpreterBridge+1032) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #65 pc 0000000000140468  /apex/com.android.runtime/lib64/libart.so (art_quick_to_interpreter_bridge+88) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.551 23418-23418/? A/DEBUG:       #66 pc 00000000009cbfb4  /system/framework/arm64/boot-framework.oat (com.android.internal.os.ZygoteInit.main+3124) (BuildId: 49cb6ce0cbaf010d2726747535cdcb7239878f72)
2020-07-03 10:30:59.552 23418-23418/? A/DEBUG:       #67 pc 00000000001375b8  /apex/com.android.runtime/lib64/libart.so (art_quick_invoke_static_stub+568) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.552 23418-23418/? A/DEBUG:       #68 pc 000000000014600c  /apex/com.android.runtime/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+276) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.552 23418-23418/? A/DEBUG:       #69 pc 00000000004b1884  /apex/com.android.runtime/lib64/libart.so (art::(anonymous namespace)::InvokeWithArgArray(art::ScopedObjectAccessAlreadyRunnable const&, art::ArtMethod*, art::(anonymous namespace)::ArgArray*, art::JValue*, char const*)+104) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.552 23418-23418/? A/DEBUG:       #70 pc 00000000004b1474  /apex/com.android.runtime/lib64/libart.so (art::InvokeWithVarArgs(art::ScopedObjectAccessAlreadyRunnable const&, _jobject*, _jmethodID*, std::__va_list)+408) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.552 23418-23418/? A/DEBUG:       #71 pc 00000000003bbb9c  /apex/com.android.runtime/lib64/libart.so (art::JNI::CallStaticVoidMethodV(_JNIEnv*, _jclass*, _jmethodID*, std::__va_list)+624) (BuildId: eddde19dbead2caefc6ed99f0476e4a5)
2020-07-03 10:30:59.552 23418-23418/? A/DEBUG:       #72 pc 00000000000ed9e4  /system/lib64/libandroid_runtime.so (_JNIEnv::CallStaticVoidMethod(_jclass*, _jmethodID*, ...)+116) (BuildId: a65bb63f108cccc2155253f97153af87)
2020-07-03 10:30:59.552 23418-23418/? A/DEBUG:       #73 pc 00000000000f0b04  /system/lib64/libandroid_runtime.so (android::AndroidRuntime::start(char const*, android::Vector<android::String8> const&, bool)+792) (BuildId: a65bb63f108cccc2155253f97153af87)
2020-07-03 10:30:59.552 23418-23418/? A/DEBUG:       #74 pc 00000000000034f8  /system/bin/app_process64 (main+1192) (BuildId: 60204f814dd23ea5d99970a48ecb4c45)
2020-07-03 10:30:59.552 23418-23418/? A/DEBUG:       #75 pc 000000000007e898  /apex/com.android.runtime/lib64/bionic/libc.so (__libc_init+108) (BuildId: 55ce0a7d78144b0290f9746ed1615719)
```

**Comment by clementetb** [comment]
We found the issue of the crash and we are working on a fix.

**Comment by cmelchior** [comment]
Should be fixed in #7094, a new Release (7.0.6) should be available in the next few days.

**Comment by GaelCO** [comment]
Nice. Thanks for the fixed.
